tests/jballerina-unit-test/src/test/resources/test-src/javainterop/negative/external_function_class_name_not_found.bal

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
```

Comment:
```suggestion
// Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```

Reply:
It seems almost all places we use `All Rights Reserved.` part in the new license header. Is there are any recent changes on it?